android/build.gradle

```diff
@@ -91,22 +90,18 @@ project.ext.versionCode = manifestVersionCode()
 project.ext.versionName = manifestVersionName()
 
 android {
-
     compileSdkVersion 28
     buildToolsVersion '28.0.3'
 
     defaultConfig {
-        versionName project.ext.versionName
+        applicationId "com.frostwire.android"
         minSdkVersion 16
-        targetSdkVersion 26
+        targetSdkVersion 28
```

Comment:
sorry, can't do this yet

Reply:
Oof, what is the reason why? Just curious :/